tests/README.md

```diff
@@ -17,7 +17,20 @@ objects at runtime.
 in the `tests` and `scripts` directories.
 
 To run the tests, follow the [setup instructions](../CONTRIBUTING.md#preparing-the-environment)
-in the `CONTRIBUTING.md` document. In particular, we recommend running with Python 3.9+.
+in the `CONTRIBUTING.md` document. In particular, you have to run with Python 3.9+.
+
+In order for `pytype_test` and `pyright_test` to work correctly, some third-party stubs
+may require dependencies external to typeshed to be installed in your virtual environment
```

Comment:
Clarify that these dependencies are in *addition* to the ones specified in `requirements-tests.txt`

```suggestion
may require extra dependencies external to typeshed to be installed in your virtual environment
```